Packs/CarbonBlackEnterpriseEDR/Integrations/CarbonBlackEnterpriseEDR/CarbonBlackEnterpriseEDR_test.py

```diff
@@ -171,3 +171,32 @@ def test_event_by_process_failing(mocker, requests_mock, demisto_args, expected_
     with pytest.raises(Exception) as e:
         client.create_search_event_by_process_request(**demisto_args)
     assert str(e.value) == expected_error_msg
+
+
+# Mock response from the API for testing
+MOCK_THREAT_TAGS_RESPONSE = {
+    'list': [
+        {'tag': 'malware'},
+        {'tag': 'suspicious'}
+    ]
+}
+
+
+def test_get_threat_tags(mocker):
```

Comment:
Please add tests for the rest of the commands you've added.
Also, for each test, please add a given, when, then documentation (can find examples from other tests in this file.)

Reply:
tests added.